tcms/testcases/static/testcases/js/search.js

```diff
@@ -118,6 +118,34 @@ $(document).ready(function() {
 
     hookIntoPagination('#resultsTable', table);
 
+    $("#check_all").click(function () {
+        if ($("input:checkbox").prop("checked")) {
```

Comment:
Is the selector inside the if condition supposed to refer to the check_all button ? If so either use the ID as selector or use event sender, etc as parameter to the callback function. If this is not what you mean then explain b/c maybe I don't quite understand.

Reply:
They selector inside of the if condition is for the rows not the check_all in the header.

Reply:
So I'm not sure why I didn't specify it down to the check_all, but it should have been. I've switched it.